### ISSUE TITLE
Paste doesn't work in PowerShell terminals when a screen reader is enabled

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#95530]
This is happening because PSReadLine is disabled intentionally

> Warning: PowerShell detected that you might be using a screen reader and has disabled PSReadLine for compatibility purposes. If you want to re-enable it, run 'Import-Module PSReadLine'.

Easy fix to use `!CONTEXT_ACCESSIBILITY_MODE_ENABLED` for the special PS paste command.

fyi @TylerLeonhardt @isidorn

**Comment by Tyriar** [comment]
To verify on Windows:

- Enable NVDA
- Copy something
- Open a terminal
- Ctrl+V should paste
